### ISSUE TITLE
Hyperparameter tuning with optuna notebook

### ISSUE THREAD

**Issue by drewoldag** [lincc-frameworks/fibad#718]
Make a common workflows notebook that shows tuning with optuna. 

Three different starting learning rates 
0.1
0.05
0.01

Three different LR schedulers. 

Default LR scheduler w/ gamma=1
Default LR scheduler w/ gamma=0.9
CosineAnnealing
